tests/models/test_cpu.py

```diff
@@ -40,6 +42,31 @@ def test_early_stopping_cpu_model(tmpdir):
     model.unfreeze()
 
 
+@pytest.mark.skipif(platform.system() == "Windows",
+                    reason="Distributed training is not supported on Windows")
+@pytest.mark.skipif((platform.system() == "Darwin" and
+                     version_parse(torch.__version__) < version_parse("1.3.0")),
```

Comment:
this is not very robust assuming pytorch may go about 10, rise `'1.3.0' > '1.10.0'`
check https://stackoverflow.com/a/11887885/4521646

Reply:
That's why I'm using the version parsing tool from the `packaging` library. It should handle things like this.

https://packaging.pypa.io/en/latest/version/